lib/whitehall_importer/import.rb

```diff
@@ -52,7 +52,7 @@ def create_document(user_ids)
         raise AbortImportError, "Document with content_id #{content_id} already exists"
       end
 
-      event = whitehall_document["editions"].first["revision_history"].select { |h| h["event"] == "create" }.first
+      event = first_edition_history.create_event
```

Comment:
Argh sorry to be a pain - I didn't notice this before. But if we use the bang equivalent here we can drop the need for the raise below this.

Reply:
I've pushed a change to reflect this